packages/content-management/table_list_view_table/src/components/table_sort_select.tsx

```diff
@@ -160,7 +160,7 @@ export function TableSortSelect({ tableSort, hasUpdatedAtMetadata, onChange }: P
       closePopover={closePopover}
       panelPaddingSize="none"
       anchorPosition="downCenter"
-      panelClassName="euiFilterGroup__popoverPanel"
+      panelProps={{ css: { width: euiTheme.base * 18 } }}
```

Comment:
nit: would prefer `18` to be a constant.  Not sure what that number is supposed to correspond to. 

@ryankeairns, you like to call this out often, what do you think?

Reply:
Y'all are super welcome to set your own constant. The number is literally what `.euiFilterGroup__popoverPanel` was setting as a width in CSS/Sass ([link](https://github.com/elastic/eui/blob/2228bef71e9c855be4a27163d763df669d12d91f/src/components/filter_group/_filter_group.scss#L37-L41)), it's just not hidden behind an EUI CSS class anymore, as we are no longer providing component-specific CSS utilities to consumers.

In any case, `width` is fairly content-specific so we would prefer consumers figure out what width they want to display based on their content and not on a generic.